### PR TITLE
remove duplicated message about unidriectional connections

### PR DESCRIPTION
**Pull request by olasaadi99**

### CODE REVIEW

File: pkg/ibmvpc/analysis_output_test.go
Comment (adisos) [inline pkg/ibmvpc/analysis_output_test.go:1]: why is this change required on the test functions?

File: pkg/ibmvpc/examples/demo_with_instances.json
Comment (adisos) [inline pkg/ibmvpc/examples/demo_with_instances.json:1]: why is the JSON output impacted by this PR?

**Review by adisos — COMMENTED** [review]
Looks good, added a few comments.

File: pkg/ibmvpc/examples/demo_with_instances.json
Comment (olasaadi99) [reply pkg/ibmvpc/examples/demo_with_instances.json:1]: because I am now comparing the after aggregate always
before the change, the comparision was after aggregate only if number of headers > 1, and there is a diff between the file with one header after aggregate and before

File: pkg/ibmvpc/examples/acl_testing3.json
Comment (adisos) [inline pkg/ibmvpc/examples/acl_testing3.json:1]: why is this file removed from the tests output files? same for `sg_testing1.json` and for `sg_testing1_new.json`?

Comment (olasaadi99) [reply pkg/ibmvpc/examples/acl_testing3.json:1]: it was deleted because there is no test for it.. (I deleted all of the generated outputs and generated new ones with the new names)

**Review by adisos — APPROVED** [review]
